### DISCUSSION TITLE
100% LHR ETH unlock with 39.6 dev ?

### DISCUSSION THREAD

**Discussion by spj3** [NebuTech/NBMiner#521] [Q&A]
Saw a post on reddit claiming to have NBMiner 39.6 dev version, with 100% ETH mining unlock on LHR (he was getting 115 mh/s with a 3080ti).
Anyone here to confirm the work in progress ?

**Comment by Franky1atp** [comment]
Hi, can anyone confirm the theory/news? That would be much appreciated!!

**Comment by madsci1016** [comment]
A PM with a dev said this was untrue. No 39.6 dev in the works right now. Another PM with the guy that posted it on reddit shows he knew he was trolling. It's fake news.
